src/parse_input.py

```diff
@@ -20,13 +20,13 @@ def valid_path(arg_path: str):
         raise argparse.ArgumentTypeError(f"{arg_path} does not exist!")
 
 
-def parse() -> tuple[str, str, bool, str]:
+def parse() -> tuple[str, str, str, bool, str]:
```

Comment:
nitpick: type declaration should be `tuple[str, str, str, bool, bool, str]` instead